src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
+#[derive(Debug, Clone)]
+pub struct HexViewConfig {
+    /// Number of bytes per line
+    pub bytes_per_line: usize,
+    /// Number of bytes per group
+    pub bytes_per_group: usize,
+    /// Separation between groups
+    pub chars_per_spacing: usize,
+    /// Separator between the address and data
+    pub addr_hex_separator: String,
+    /// Separator between the hex and ascii representations
```

Comment:
Controls the separator between the hex output and the ASCII representation of the daja.

Default is `: `